worker/Dockerfile

```diff
@@ -1,4 +1,3 @@
-# syntax=docker/dockerfile:1
```

Comment:
@kirisaki 
Please don't delete things that are not necessary!

Reply:
What do you mean?

Reply:
@kirisaki  Any reason why this line was deleted? Please retain the parser directive on at the beginning of the Dockerfile.

Reply:
I deleted it because VS Code pointed out it. I revert it.

Reply:
Done.

Reply:
Thank you so much 🙇